nursery/reference-screen-saver-executable.yml

```diff
@@ -1,19 +1,19 @@
 rule:
   meta:
-    name: reference screen saver executable
+    name: persist via screensaver registry key
```

Comment:
I don't fully understand what you are saying, sorry.

How I see it:
The old rule looks for the "SCRNSAVE.EXE" string and then labels it as persistence (or at least it is in the persistence namespace. To my understanding that should imply persistence). Only looking for the string is of course not very accurate, as it could result in false positives.
With the proposed change, it matches only writes to the specific location. This would reduce false positives, and should not result in new false negatives.

Why would you prefer the old rule over the new one?
The rule is not very prevalent indeed, but it is still a working persistence technique used by malware.

Thanks in advance for helping me understand :)

Reply:
My thinking was that the rule is "good enough" so we don't have to make it more specific and the more general rule name describes the concrete observable here. I see your points though. @mike-hunhoff, what do you think?

Reply:
or @williballenthin any thoughts here? :)

Reply:
I agree with @jorik-utwente points and think we should roll with the updates proposed here. Overall, these changes make the rule more specific to its original intention.

Reply:
this works for me as well